src/libmongoc/tests/test-mongoc-cursor.c

```diff
@@ -316,10 +321,11 @@ _test_common_opts (void *ctx)
    mongoc_cursor_set_max_await_time_ms (cursor, 3);
    ASSERT_CMPINT (mongoc_cursor_get_max_await_time_ms (cursor), ==, 3);
    /* prime the cursor. */
-   BSON_ASSERT (mongoc_cursor_next (cursor, &doc));
+   ASSERT_OR_PRINT (mongoc_cursor_next (cursor, &doc), cursor->error);
```

Comment:
This helped debug the issue. If this fails again, we'd definitely want the cursor's error message.

Reply:
+1